content/en/docs/tasks/traffic-management/ingress/ingress-sidecar-tls-termination/index.md

```diff
@@ -0,0 +1,307 @@
+$ kubectl -n test create secret generic httpbin-mtls-termination-cacert --from-file=ca.crt=./example.com.crt
+$ kubectl -n test create secret tls httpbin-mtls-termination --cert ./httpbin.test.svc.cluster.local.crt --key ./httpbin.test.svc.cluster.local.key
+{{< /text >}}
+
+## Create httpbin deployment and services
+
+When the httpbin deployment is to be created, we need to use `userVolumeMount` annotations in the deployment to make sure the certificates are mounted to the istio-proxy sidecar.
+Note: This step is needed until Istio supports `credentialName` for sidecar configuration.
+
+{{< text yaml >}}
+sidecar.istio.io/userVolume: '{"tls-secret":{"secret":{"secretName":"httpbin-mtls-termination","optional":true}},"tls-ca-secret":{"secret":{"secretName":"httpbin-mtls-termination-cacert"}}}'
+sidecar.istio.io/userVolumeMount: '{"tls-secret":{"mountPath":"/etc/istio/tls-certs/","readOnly":true},"tls-ca-secret":{"mountPath":"/etc/istio/tls-ca-certs/","readOnly":true}}'
+{{< /text >}}
+
+The complete deployment yaml using `userVolumeMount` and service configuration for `httpbin` can be found below:
+
+{{< text bash >}}
+$ kubectl -n test apply -f - <<EOF
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: httpbin
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: httpbin
+  labels:
+    app: httpbin
+    service: httpbin
+spec:
+  ports:
+  - port: 8443
+    name: https
+    targetPort: 9080
+  - port: 8080
+    name: http
+    targetPort: 9081
+  selector:
+    app: httpbin
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: httpbin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: httpbin
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: httpbin
+        version: v1
+      annotations:
+        sidecar.istio.io/userVolume: '{"tls-secret":{"secret":{"secretName":"httpbin-mtls-termination","optional":true}},"tls-ca-secret":{"secret":{"secretName":"httpbin-mtls-termination-cacert"}}}'
+        sidecar.istio.io/userVolumeMount: '{"tls-secret":{"mountPath":"/etc/istio/tls-certs/","readOnly":true},"tls-ca-secret":{"mountPath":"/etc/istio/tls-ca-certs/","readOnly":true}}'
+    spec:
+      serviceAccountName: httpbin
+      containers:
+      - image: docker.io/kennethreitz/httpbin
+        imagePullPolicy: IfNotPresent
+        name: httpbin
+        ports:
+        - containerPort: 80
+EOF
+{{< /text >}}
+
+## Create sidecar configuration for httpbin to enable external mTLS on ingress
+
+This is the core step in this feature, where `sidecar` API is used to configure the ingress TLS settings.
+
+{{< text bash >}}
+$ kubectl -n test apply -f - <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: Sidecar
+metadata:
+  name: ingress-sidecar
+  namespace: test
+spec:
+  workloadSelector:
+    labels:
+      app: httpbin
+      version: v1
+  ingress:
+  - port:
+      number: 9080
+      protocol: HTTPS
+      name: external
+    defaultEndpoint: 0.0.0.0:80
+    tls:
+      mode: MUTUAL
```

Comment:
So we still support MUTUAL, SIMPLE, it is better to mention that.

Reply:
Added